### DISCUSSION TITLE
Italian traduction

### DISCUSSION THREAD

**Discussion by simone7121** [kaiserdj/Stickerworld#32] [Show and tell]
Hello @kaiserdj 
I've made an Italian file for the bot.
You can find it attached to the discussion.
Greetings !
[it.zip](https://github.com/kaiserdj/Stickerworld/files/6118975/it.zip)

**Comment by kaiserdj** [comment]
If you want you can make a pull requests, so that you leave as a collaborator in the project.

Also thank you very much
